test/startservers.py

```diff
@@ -77,32 +77,37 @@ def start(race_detection, fakeclock=None, account_uri=None):
-        [9380, 'mail-test-srv --closeFirst 5 --cert test/mail-test-srv/localhost/cert.pem --key test/mail-test-srv/localhost/key.pem'],
-        [8005, 'ocsp-responder --config %s' % os.path.join(default_config_dir, "ocsp-responder.json")],
+        [53, './bin/sd-test-srv --listen :53'], # Service discovery DNS server
+        [8003, './bin/boulder-sa --config %s --addr sa1.boulder:9095 --debug-addr :8003' % os.path.join(default_config_dir, "sa.json")],
+        [8103, './bin/boulder-sa --config %s --addr sa2.boulder:9095 --debug-addr :8103' % os.path.join(default_config_dir, "sa.json")],
+        [4500, './bin/ct-test-srv --config test/ct-test-srv/ct-test-srv.json'],
+        [8009, './bin/boulder-publisher --config %s --addr publisher1.boulder:9091 --debug-addr :8009' % os.path.join(default_config_dir, "publisher.json")],
+        [8109, './bin/boulder-publisher --config %s --addr publisher2.boulder:9091 --debug-addr :8109' % os.path.join(default_config_dir, "publisher.json")],
+        [9380, './bin/mail-test-srv --closeFirst 5 --cert test/mail-test-srv/localhost/cert.pem --key test/mail-test-srv/localhost/key.pem'],
+        [8005, './bin/ocsp-responder --config %s' % os.path.join(default_config_dir, "ocsp-responder.json")],
         # The gsb-test-srv needs to be started before the VA or its intial DB
         # update will fail and all subsequent lookups will be invalid
-        [6000, 'gsb-test-srv -apikey my-voice-is-my-passport'],
-        [8053, 'challtestsrv --dns01 :8053,:8054 --management :8055 --http01 :5002 --tlsalpn01 :5001'],
-        [8004, 'boulder-va --config %s --addr va1.boulder:9092 --debug-addr :8004' % os.path.join(default_config_dir, "va.json")],
-        [8104, 'boulder-va --config %s --addr va2.boulder:9092 --debug-addr :8104' % os.path.join(default_config_dir, "va.json")],
-        [8001, 'boulder-ca --config %s --ca-addr ca1.boulder:9093 --ocsp-addr ca1.boulder:9096 --debug-addr :8001' % os.path.join(default_config_dir, "ca-a.json")],
-        [8101, 'boulder-ca --config %s --ca-addr ca2.boulder:9093 --ocsp-addr ca2.boulder:9096 --debug-addr :8101' % os.path.join(default_config_dir, "ca-b.json")],
-        [6789, 'akamai-test-srv --listen localhost:6789 --secret its-a-secret'],
-        [8006, 'ocsp-updater --config %s' % os.path.join(default_config_dir, "ocsp-updater.json")],
-        [8002, 'boulder-ra --config %s --addr ra1.boulder:9094 --debug-addr :8002' % os.path.join(default_config_dir, "ra.json")],
-        [8102, 'boulder-ra --config %s --addr ra2.boulder:9094 --debug-addr :8102' % os.path.join(default_config_dir, "ra.json")],
-        [4431, 'boulder-wfe2 --config %s' % os.path.join(default_config_dir, "wfe2.json")],
-        [4000, 'boulder-wfe --config %s' % os.path.join(default_config_dir, "wfe.json")],
+        [6000, './bin/gsb-test-srv -apikey my-voice-is-my-passport'],
+        # NOTE(@cpu): We specify explicit bind addresses for -https01 and
```

Comment:
```suggestion
        # NOTE(@cpu): We specify explicit bind addresses for -http01 and
```

Reply:
In this case it isn't a typo there are `-http01` and -https01` args for the challtestsrv.  One for the HTTP interface, one for HTTPS.

Overall the names of the `pebble-challtestsrv` CLI args could probably be updated to be clearer (e.g. `-dns01` is used for both ACME DNS-01 TXTs and more broadly for mock DNS) but that would be an issue to address in Pebble & update here.

Does that make sense?